workspaces/global-header/plugins/global-header/src/components/MenuItemLink/MenuItemLinkContent.tsx

```diff
@@ -13,64 +13,79 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import React from 'react';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import { HeaderIcon } from '../HeaderIcon/HeaderIcon';
 import { useTheme } from '@mui/material/styles';
+import LaunchIcon from '@mui/icons-material/Launch';
 
 interface MenuItemLinkContentProps {
   icon?: string;
   label?: string;
   subLabel?: string;
+  isExternalLink?: boolean;
 }
 
 export const MenuItemLinkContent: React.FC<MenuItemLinkContentProps> = ({
   icon,
   label,
   subLabel,
+  isExternalLink = false,
 }) => {
   const theme = useTheme();
   return (
     <Box
       sx={{
         display: 'flex',
         alignItems: 'center',
+        justifyContent: 'space-between',
         margin: '8px 0',
         color: 'inherit',
+        width: '100%',
       }}
     >
-      {icon && (
-        <HeaderIcon
-          icon={icon}
-          size="small"
-          layout={
-            label
-              ? {
-                  marginRight: '0.5rem',
-                  flexShrink: 0,
-                  color:
-                    theme.palette.mode === 'dark'
-                      ? theme.palette.text.primary
-                      : theme.palette.text.disabled,
-                }
-              : {}
-          }
-        />
-      )}
-      <Box>
-        {label && (
-          <Typography variant="body2" color={theme.palette.text.primary}>
-            {label}
-          </Typography>
-        )}
-        {subLabel && (
-          <Typography variant="caption" color="text.secondary">
-            {subLabel}
-          </Typography>
+      <Box display="flex" alignItems="center">
+        {icon && (
+          <HeaderIcon
```

Comment:
I'm not sure why but the icons here look misaligned to my eye:

![image](https://github.com/user-attachments/assets/6d443deb-0e95-4987-92f0-186e230aefbb)

can you confirm whether this is because some `app-config` misconfiguration, or whether the icons should be aligned like this?

Reply:
Fixed it. 
<img width="203" alt="Screenshot 2025-03-25 at 2 22 53 AM" src="https://github.com/user-attachments/assets/700ae3c6-727d-47e0-b9f3-32ad3fa80fe1" />